modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectViewLocalServiceImpl.java

```diff
@@ -286,15 +301,67 @@ private List<ObjectViewSortColumn> _addObjectViewSortColumns(
 			});
 	}
 
-	private void _validate(
+	private void _validate(long objectViewId, long objectDefinitionId)
+		throws PortalException {
+
+		ObjectView objectView = objectViewPersistence.fetchByODI_DOV_First(
+			objectDefinitionId, true, null);
+
+		if ((objectView != null) &&
+			(objectView.getObjectViewId() != objectViewId)) {
+
+			throw new DefaultObjectViewException(
+				"There can only be one default object view");
+		}
+	}
+
+	private void _validateObjectViewColumn(
+			ObjectView objectView, List<ObjectViewColumn> objectViewColumns)
+		throws ObjectViewColumnFieldNameException {
+
+		List<ObjectField> objectFields =
+			_objectFieldPersistence.findByObjectDefinitionId(
+				objectView.getObjectDefinitionId());
+
+		Set<String> objectFieldNames = new LinkedHashSet<>();
+
+		objectFieldNames.addAll(
+			SetUtil.fromArray(
+				"creator", "dateCreated", "dateModified", "id", "status"));
```

Comment:
Could you initialize the variable with the SetUil.fromArray(), please?
It would look like this:
`  Set<String> objectFieldNames = SetUtil.fromArray("creator", "dateCreated", "dateModified", "id", "status");`